haystack/nodes/prompt/prompt_node.py

```diff
@@ -687,10 +702,10 @@ def __call__(self, *args, **kwargs) -> List[Any]:
             sa(documents=[Document("I am in love and I feel great!")])
         ```
         """
-        if "prompt_template_name" in kwargs:
-            prompt_template_name = kwargs["prompt_template_name"]
-            kwargs.pop("prompt_template_name")
-            return self.prompt(prompt_template_name, *args, **kwargs)
+        if "prompt_template" in kwargs:
+            prompt_template = kwargs["prompt_template"]
+            kwargs.pop("prompt_template")
+            return self.prompt(prompt_template, *args, **kwargs)
```

Comment:
Let's keep it simple and just throw anything that selects the prompt template into `prompt_template`